oneiromancy/index.test.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-env node, jest */
+
+jest.mock('../lib/openai', () => ({
+	chat: {
+		completions: {
+			create: jest.fn(() => Promise.resolve({
+				choices: [{
+					message: {
+						content: '今日の運勢は【80点】です。良い一日になりそうです。',
+					},
+				}],
+			})),
+		},
+	},
+}));
+
+jest.mock('../achievements');
+jest.mock('fs/promises', () => ({
+	readFile: jest.fn(() => Promise.resolve(Buffer.from(`
+- role: system
+  content: あなたは夢占いBOTです。
+`))),
+}));
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const {default: Slack} = require('../lib/slackMock.ts');
```

Comment:
Use `import` instead of `require`. You can disable `import-first` eslint rules. Refer the test files in helloworld directory.